src/components/common/TravelTipsCard/TravelTipsCard.jsx

```diff
@@ -0,0 +1,77 @@
+import { memo, useState } from 'react';
+import { cn } from '../../../utilities/cn';
+import { Link } from 'react-router-dom';
+import TravelTipsStub from '../../../assets/img/TravelTips/TravelTipsStub.png';
+import Heart from './../../../assets/img/TravelTips/heart.svg';
+import ReadMore from './../../../assets/img/TravelTips/read_more.svg';
+
+const TravelTipsCard = ({ tipHeader, tipText, date, image }) => {
+  const [likesAmount, setLikesAmount] = useState(0);
+  const [isLiked, setIsLiked] = useState(false);
+
+  const handleLikeClick = e => {
+    e.preventDefault();
+    setIsLiked(prevLiked => {
+      setLikesAmount(prev => (prevLiked ? prev - 1 : prev + 1));
```

Comment:
updater function should be pure 
https://react.dev/reference/react/useState#my-initializer-or-updater-function-runs-twice

In your case, you update another state inside an updater fn.